dbt/adapters/snowflake/relation_configs/dynamic_table.py

```diff
@@ -107,7 +87,7 @@ def requires_full_refresh(self) -> bool:
 
 @dataclass
 class SnowflakeDynamicTableConfigChangeset:
-    target_lag: Optional[SnowflakeDynamicTableTargetLagConfigChange] = None
+    target_lag: Optional[SnowflakeDynamicTableWarehouseConfigChange] = None
```

Comment:
This would be a change that looks like `SnowflakeDynamicTableWarehouseConfigChange`, but I would keep it separate, so `SnowflakeDynamicTableTargetLagConfigChange`. The class would be almost identical to the warehouse one. Leave it in this file, just like the warehouse one.

Reply:
done